src/services/api/user-feedback-api/comments.ts

```diff
@@ -33,32 +35,37 @@ export const commentsApi = createApi({
         body: post
       }),
       invalidatesTags: (_result, _error, arg) => [
-        { type: 'Comments', id: arg.id }
+        { type: 'Comments', id: `${arg.id}-1` }
       ]
     }),
 
     updateComment: builder.mutation<
       CommunityPost,
-      { id: string; post: Partial<CommunityPost> }
+      { id: string; post: Partial<CommunityPost>; page: number }
     >({
       query: ({ id, post }) => ({
-        url: `/thread/${id}/${post.pid}`,
+        url: `/thread/${id}/${post.pid}?postIndex=${post.index}`,
```

Comment:
Må man ha med post.index som parameter for å oppdatere en post, følger ikke api-et til nodebb rest?

Reply:
Må ha det med for å oppdatere eller slette en post som ikke på første side, så må i så fall følge med på hvilken side den kom fra, eventuelt om den har post.index 20 eller over. Dunno, blir noe conditional-greier som virker unødvendig, kan vel like så greit alltid ta med post.index?

Reply:
Må ha det med for å oppdatere eller slette en post som ikke på første side, så må i så fall følge med på hvilken side den kom fra, eventuelt om den har post.index 20 eller over. Dunno, blir noe conditional-greier som virker unødvendig, kan vel like så greit alltid ta med post.index?